lib/jekyll-last-modified-at/hook.rb

```diff
@@ -6,11 +6,21 @@ module Hook
       def self.add_determinator_proc
         proc { |item|
           format = item.site.config.dig('last-modified-at', 'date-format')
-          item.data['last_modified_at'] = Determinator.new(item.site.source, item.path,
-                                                           format)
+          use_git_cache = item.site.config.dig('last-modified-at', 'use-git-cache')
+          item.data['last_modified_at'] = Determinator.new(item.site.source, item.relative_path,
+                                                           format, use_git_cache)
         }
       end
 
+      Jekyll::Hooks.register :site, :after_reset do |site|
```

Comment:
I personally didn't find this hook very helpful: it triggers a cache clear every file change while I'm developing, slowing my incremental builds.